core/storage/base_model/gae_models_test.py

```diff
@@ -111,6 +111,60 @@ class TestVersionedModel(base_models.VersionedModel):
     SNAPSHOT_CONTENT_CLASS = TestSnapshotContentModel
 
 
+class TestCommitLogEntryModel(base_models.BaseCommitLogEntryModel):
+    @classmethod
+    def _get_instance_id(cls, instance_id, version):  # pylint: disable=unused-argument
+        """A dummy function to mimic the actual _get_instance_id. In this case,
+        the actual id itself is the argument with this function just returning
```

Comment:
Why not just concat with version in the usual way (with a dummy name at the front)? More realistic and avoids possible shadowing of errors since the two types of IDs would be distinct.

Reply:
Yeah, changed the function to return a dummy id.